embedded-localstack/src/test/java/com/playtika/test/localstack/DisabledLocalStackTest.java

```diff
@@ -0,0 +1,33 @@
+package com.playtika.test.localstack;
+
+
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.config.ConfigurableListableBeanFactory;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.testcontainers.containers.localstack.LocalStackContainer;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringRunner.class)
```

Comment:
It is redundant here. `@SpringBootTest` already includes `@ExtendWith(SpringExtension)`

Reply:
Fixed.